lstmcpipe/utils.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+import subprocess
+
+
+def rerun_cmd(cmd, outfile, max_ntry=2):
```

Comment:
Don't you want to add test for this, @vuillaut ? 😇

Reply:
👀

Reply:
done